pkg/scheduler/metrics/metrics.go

```diff
@@ -262,6 +265,9 @@ func Register() {
 			legacyregistry.MustRegister(metric)
 		}
 		volumeschedulingmetrics.RegisterVolumeSchedulingMetrics()
+		PodScheduleSuccesses = scheduleAttempts.With(metrics.Labels{"result": "scheduled"})
```

Comment:
instead of doing this, perhaps we should follow the pendingPods metric approach, define functions that return the metric (see lines 275-292)

Reply:
ok, I guess we access those handlers directly in the scheduler, ok I guess this is fine.